datastore/mongodb/filter_outputs.go

```diff
@@ -0,0 +1,27 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-cantabular-filter-flex-api/model"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+// CreateFilterOutput creates a new FilterOutputs in the CantabularFilters colllection
+func (c *Client) CreateFilterOutput(ctx context.Context, f *model.FilterOutput) error {
+	id, err := c.generate.UUID()
+	if err != nil {
+		return errors.Wrap(err, "failed to generate UUID: %w")
+	}
+
+	f.ID = id.String()
+
+	col := c.collections.filterOutputs
+
+	if _, err := c.conn.Collection(col.name).UpsertById(ctx, f.ID, bson.M{"$set": f}); err != nil {
```

Comment:
We need to check that the item was created here. If it was not (e.g. something got updated), that is an error

Reply:
I disagree with not inspecting the output but happy to accept your conclusion